Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsOverlayUnderlayScreen.mm

```diff
@@ -552,6 +576,16 @@ - (void) onShowSwitchChanged:(id)sender
     }
 }
 
+- (void) showLabels:(BOOL)show
+{
+    [_settings.keepMapLabelsVisible set:show];
+    //refresh layers
+//    _app.data.overlayAlpha = _app.data.overlayAlpha;
+//    _app.data.underlayAlpha = _app.data.underlayAlpha;
+    [_app.data refreshOverlayUnderlayAlpha];
+//    [[[OsmAndApp instance] mapSettingsChangeObservable] notifyEvent];
```

Comment:
1 - Do not commit comments. Looks like draft code.
2 - Looks like it was right code.